src/main/interface/cli.c

```diff
@@ -4478,6 +4650,8 @@ const clicmd_t cmdTable[] = {
     CLI_COMMAND_DEF("servo", "configure servos", NULL, cliServo),
 #endif
     CLI_COMMAND_DEF("set", "change setting", "[<name>=<value>]", cliSet),
+    CLI_COMMAND_DEF("osdprofile", "set display pages for OSD element", "[<name>=<value>]", cliOSDProfile),
+    CLI_COMMAND_DEF("osddbg", "show osd profile value", NULL, cliOSDDbg),
```

Comment:
Not sure that `osddbg` is the right name. In general, `osdprofile` without arguments should show the current OSD profile selection, and with only a number it should select the OSD profile.

Reply:
osddbg Was actually just for debugging.  I have now removed it.  osdprofile without arguments now prints the current OSD profile selection and with a number selects the OSD profile.  I guess this has the added benefit that you can configure osd profiles, not use adjustment ranges and just set the profile to the one you want.